tests/pull-test.sh

```diff
@@ -63,12 +63,28 @@ $OSTREE --repo=ostree-srv/gnomerepo checkout main checkout-origin-main
 echo moomoo > checkout-origin-main/baz/cow
 ${CMD_PREFIX} ostree --repo=ostree-srv/gnomerepo commit -b main -s "" --tree=dir=checkout-origin-main
 ${CMD_PREFIX} ostree --repo=ostree-srv/gnomerepo static-delta generate main
+${CMD_PREFIX} ostree --repo=ostree-srv/gnomerepo summary -u
```

Comment:
Is there a reason why this one needed to be added? Seems like the new logic here shouldn't affect it.